examples/births-deaths/plot_output.R

```diff
@@ -0,0 +1,31 @@
+library(tidyverse)
+library(jsonlite)
+## Todo:
+## - Plot population changes
+## - Plot SIR
+## - Compare with theoretical foi with population change
+dir <- file.path("examples", "births-deaths")
+params <- read_json(file.path(dir, "input.json"))
+population <- params$population
+age_groups <- params$foi_groups
+
+
+
+output_df <- read_csv(file.path(dir, "incidence.csv")) |>
+  dplyr::filter(infection_status == "I") |>
+  group_by(time, age_group) |>
+  mutate(inf = n()) |>
```

Comment:
this should be a summarize. Imagine you have two infections that occur at the same time -- you want to count those each once, not twice.

Reply:
Yeah that makes sense. Though, unlikely with this approach.